src/main/java/com/abcdedu_backend/contact/entity/ContactType.java

```diff
@@ -1,14 +1,26 @@
 package com.abcdedu_backend.contact.entity;
 
+import com.abcdedu_backend.exception.ApplicationException;
+import com.abcdedu_backend.exception.ErrorCode;
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 
+import java.util.stream.Stream;
+
 @Getter
 @RequiredArgsConstructor
 public enum ContactType {
-    CLASS("강의"),
-    TRAINING("교사연수"),
-    ETC("기타");
+    TRAINING("training"),
+    CLASS("class"),
+    ETC("etx");
```

Comment:
p5: 사소한건데, 오타인가 해서 말씀드려요
```suggestion
    CLASS("class"),
    ETC("etc");
```

Reply:
휴우우우!!!!!!! 발견해주셔서 감사합니다.. 바로 고쳐서 merge 합니다ㅠㅠ!